{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{cookiecutter.project_name}}/monitoring/SetUpDataMonitoringForInferenceTable.py

```diff
@@ -0,0 +1,135 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC
+# MAGIC # Data Monitoring for Inference Table
+# MAGIC see monitoring/README.md for details
+
+# COMMAND ----------
+
+# Install the Data Monitoring client library
+# TODO: Fill in wheel URL
+%pip install "{wheel_URL}"
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC 
+# MAGIC ### Required Fields
+# MAGIC TODO: Fill in and review required fields
+
+# COMMAND ----------
+
+# The name of the column that represents the inference timestamp (i.e. time of prediction) for this row. This column should be of TimestampType.
+# By default, BatchInference notebook generates the column with name `inference_timestamp`.
+timestamp_col = "inference_timestamp"
+
+# The name of the column that represents the model version used to predict on this row. 
+# This should correspond to the MLflow registered model version (or any identifier unique to a specific trained model).
+# By default, BatchInference notebook generates the column with name `model_version`.
+model_version_col = "model_version"
+
+# The name of the column that holds the prediction (from inference) for this row.
+# By default, BatchInference notebook generates the column with name `prediction`.
+prediction_col = "prediction"
+
+# The type of ML problem that the model is solving. This should be either “classification” or “regression”.
+# Other problem types are not supported at this time. 
+problem_type = "regression"
+
+# The inference table holding data to be monitored. Table name format {catalog}.{schema}.{table}
+inference_table_name = "{catalog}.{schema}.{table}"
+
+# Name of the schema in which to create output tables. format {catalog}.{schema}
+output_schema_name = "{catalog}.{schema}"
```

Comment:
I see what you're saying about the batch inference notebook generating default column names. However, these parameters still require intervention?
- `regression`: if the user accidentally creates the monitor with regression without realizing they had to update this to classification, they'll need to manually delete and recreate
- `inference_table_name` and `output_schema_name`: these must be changed because the bracket defaults are not actual table names.

Should we make the defaults `None` and then fail the notebook if the user has not changed them?

Reply:
Thank you! It makes sense. I'll update it.